setup.py

```diff
@@ -43,6 +47,7 @@ def readme():
     test_suite='tests',
     extras_require={
         'currency': currency_requires,
+        'geo': geo_requires,
```

Comment:
We might want to name these `country` and `country_requires`, since the rest of the geo fields (latitude and longitude) don't require this.